contracts/vaults/balancer/BalancerVaultTypes.sol

```diff
@@ -188,8 +188,6 @@ struct StrategyVaultSettings {
     uint16 balancerOracleWeight;
     /// @notice Cool down in minutes for normal settlement
     uint16 settlementCoolDownInMinutes;
-    /// @notice Cool down in minutes for post maturity settlement
-    uint16 postMaturitySettlementCoolDownInMinutes;
```

Comment:
Since you are removing these from storage, when you upgrade your vault on goerli it will mess up all the settings. You will need to reset the settings manually in order to get them to work properly.

Reply:
actually, postMaturitySettlementCoolDownInMinutes was already removed when I deployed the contracts on goerli